src/codeu/chat/client/simplegui2/AlertBox.java

```diff
@@ -0,0 +1,63 @@
+package codeu.chat.client.simplegui2;
+
+import codeu.chat.client.ClientContext;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.layout.VBox;
+import javafx.stage.Modality;
+import javafx.stage.Stage;
+
+/**
+ * Created by noramyer on 4/6/17.
+ * When called, creates a pop-up window displaying message that must be exited before user can continue to use program
+ */
+public final class AlertBox {
+
+    private Stage window;
+    private Label label1;
+    private Button closeButton;
+
+    public AlertBox(String title, String message){
+
+        this.window = new Stage();
+        initialize(title, message);
```

Comment:
I'm not a fan of the initialize functions if they are only ever called in the constructor.